File: activityTestCase/app/src/androidTest/java/com/example/activitytestcase/MainActivityTest.java
package com.example.activitytestcase;

import android.widget.EditText;
import android.widget.TextView;

import androidx.test.rule.ActivityTestRule;

import org.junit.After;
import org.junit.Before;
import org.junit.Rule;
import org.junit.Test;

import static org.junit.Assert.*;

public class MainActivityTest {

    private MainActivity mainActivity;
    private TextView textView;
    private EditText editText;

    @Rule
    //launch the activity
    public ActivityTestRule<MainActivity> activityActivityTestRule = new ActivityTestRule<>(MainActivity.class);

    @Before
    public void setUp() throws Exception {
        //get the activity
        mainActivity= activityActivityTestRule.getActivity();
        textView = mainActivity.findViewById(R.id.textone);
    }

    @Test
    public void doingTest()
    {
        editText = mainActivity.findViewById(R.id.editone);
      //  assertEquals("Ashish",textView.getText().toString());
        assertEquals("Ranjan",editText.getText().toString());
    }


    @After
    public void tearDown() throws Exception {
        mainActivity = null;
    }
}

File: activityTestKotlin/app/src/androidTest/java/com/example/activitytestkotlin/MainActivityTest.kt
package com.example.activitytestkotlin

import android.widget.TextView
import androidx.test.rule.ActivityTestRule
import org.junit.After
import org.junit.Before

import org.junit.Assert.*
import org.junit.Rule
import org.junit.Test

class MainActivityTest {

    @Rule
    @JvmField
    //launch the activity
    var activityTestRule = ActivityTestRule(MainActivity::class.java)


    lateinit var activity: MainActivity

    @Before
    fun setUp() {
         //get the activity
        activity = activityTestRule.activity
       // var textview:TextView = activity.findViewById(R.id.textviewname)

    }

    @Test
    fun testactivity(){
        //assertEquals("Ashish", textview1.text.toString())
        //assertEquals("Ashish",textview.text.toString())

      //   var edit:TextView = activity.findViewById(R.id.textviewname);
        // var s = edit.text.toString();
         //assertEquals("Ashish",s)

        assertEquals("Ashish Ranjan", activity.add("Ashish ","Ranjan"))

    }



    @After
    fun tearDown() {

    }
}